Solutions/Marain.Workflow.Abstractions/Marain/Workflows/InvokeExternalServiceAction.cs

```diff
@@ -161,7 +161,7 @@ private async Task ExecuteInternalAsync(WorkflowInstance instance, IWorkflowTrig
             else
             {
                 this.logger.LogDebug(
-                    "Request to external URL '{externalUrl}' did not return any response.",
+                    "Request work workflow instance '{workflowInstanceId}' to external URL '{externalUrl}' did not return any response.",
```

Comment:
I'm not sure this message is correctly formatted?

Reply:
I, on the other hand, am certain that it is not. Oops. Fixed.